cli/src/semgrep/formatter/text.py

```diff
@@ -586,7 +586,7 @@ def lang_of_path(path: str) -> str:
         )
 
     console.print(f"Slowest {items_to_show} rules to match")
-    slowest_rule_times = sorted(rule_match_timings.items(), reverse=True)[
+    slowest_rule_times = sorted(rule_match_timings.items(), reverse=True, , key=lambda x: x[1])[
```

Comment:
```suggestion
    slowest_rule_times = sorted(rule_match_timings.items(), reverse=True, key=lambda x: x[1])[
```

Reply:
:facepalm: thank you, I'll fix that asap